src/lib/wallet/GoodWalletProvider.js

```diff
@@ -239,7 +239,18 @@ export const GoodWalletProvider = ({ children, disableLoginAndWatch = false }) =
 
         global.userStorage = storage
         global.wallet = wallet
-        setWalletAndStorage({ goodWallet: wallet, userStorage: storage, celowallet, fusewallet, web3Provider })
+
+        if (logMethod) {
```

Comment:
so we update user profile with the last login method he/she used, right ?

Reply:
So basically the logMethod gets set on signUp

but for existing users we have to add this flow somewhere.
so basically yes, last login method (Is it possible to change login method without deleting account?)

Reply:
this gets fired upon returning from provider X before/during preparing wallet